Resources/Public/css/grid.css

```diff
@@ -1,4 +1,4 @@
-.flux-grid { width: 100%; }
+.flux-grid { width: 100%; table-layout:fixed;}
```

Comment:
THANK YOU! Seriously, thank you so much for finding this one. I couldn't figure out why my 2/3 column sizing was rendering as 50/50. Excellent, please create this as a separate pull request.  :)

If you do, please add to css as well:

```
.t3-is-dragged .t3-page-ce-body {
	max-height: none;
}
```

That's the hight-limit-overflow-hidden prevention for LTS.